atlascope/core/rest/endpoints/job_endpoints.py

```diff
@@ -20,7 +19,7 @@ class JobViewSet(
 ):
     model = Job
     queryset = Job.objects.all().order_by('id')
-    permission_classes = [IsAuthenticated]
+    permission_classes = []
```

Comment:
You'll actually need this in the other viewsets as well. The reason that you were only getting 401s for job endpoints is because those tests are the only "write" endpoints we test. If we add a test for creating a dataset, the same thing would happen to that test without this line in `dataset_endpoints.py`

Reply:
Instead of doing that, I figured out the configuration option to disable permission management in all routes: fdedfe99cb8acab7b5e40a4e79364635b799d0a5.